tests/test_consumer.py

```diff
@@ -116,6 +116,19 @@ async def test_simple_consumer(self):
         # will ignore, no exception expected
         await consumer.stop()
 
+    def test_create_producer_no_running_loop(self):
```

Comment:
Typo, should be `test_create_consumer_no_running_loop `, not `producer`

Reply:
Thanks, fixed